lib/jekyll/commands/build.rb

```diff
@@ -71,12 +71,23 @@ def build(site, options)
         #
         # Returns nothing.
         def watch(site, options)
-          External.require_with_graceful_fail "jekyll-watch"
-          watch_method = Jekyll::Watcher.method(:watch)
-          if watch_method.parameters.size == 1
-            watch_method.call(options)
+          if Utils::Platforms.windows?
+            Jekyll.logger.warn "", "--watch unsupported on Windows. "
```

Comment:
Date: Fri, 12 Aug 2016 07:28:39 -0700
From: notifications@github.com
To: jekyll@noreply.github.com
CC: cori.schlegel@gmail.com; team_mention@noreply.github.com
Subject: Re: [jekyll/jekyll] Fix #5233: Increase our ability to detect Windows. (#5235)

In lib/jekyll/commands/build.rb:

> @@ -71,12 +71,23 @@ def build(site, options)
>          #
>          # Returns nothing.
>          def watch(site, options)
> -          External.require_with_graceful_fail "jekyll-watch"
> -          watch_method = Jekyll::Watcher.method(:watch)
> -          if watch_method.parameters.size == 1
> -            watch_method.call(options)
> +          if Utils::Platforms.windows?
> +            Jekyll.logger.warn "", "--watch unsupported on Windows. "

Maybe "The --watch flag is unsupported..." for a bit of polish.


—
You are receiving this because you are on a team that was mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.